### PR TITLE
Add sponsorship section

### PR DESCRIPTION
**Pull request by isabelledelmundo**
### Tickets:
- HCK-10

### List of changes:
- Added Sponsorship Section of static site

### Type of change:

- [x] New feature (non-breaking change which adds functionality)

### How did you do this?
Created sponsorship section styled components based on the about section and applied it to the graphic and text in the section. 

### Why are you choosing this approach?
To make it look uniform

### Questions for code reviewers?
N/A

### PR Checklist:
- [x] Merged `develop` branch (before testing)
- [x] Linted my code locally
- [x] Tested all links in project relevant browsers
- [x] Tested all links on different screen sizes
- [x] Referenced all useful info (issues, tasks, etc)

### CODE REVIEW

File: src/components/hero.js [hunk 1]
```diff
@@ -6,15 +6,14 @@ import * as Styles from "../styles/components/index.js"
...
-            <h1 class="hero__heading">McHacks</h1> 
+        <div className='hero__copy'>
+            <h2 className="hero__subheading">Youpi doop firggin slogan:</h2> 
+            <h1 className="hero__heading">McHacks</h1> 
             <p>McGill University    Feb 1-3, 2020</p>
             <p>Sign up for updates on McHacks. No spam, we promise.</p>
             <button>your email here -></button>
         </div>
     </Styles.Hero>
-
```
Comment (ashaduzzamane) [inline src/components/hero.js]: good job condensing code :)

File: src/styles/components/index.js [hunk 2]
```diff
@@ -142,9 +146,72 @@ export const AboutImg = styled.img`
         padding: 0 2rem;
     }
 `
+export const SponsContent = styled.div`
```
Comment (loreina) [inline src/styles/components/index.js]: can we update these variables to be `Spon-` instead? I think we're gonna unify `spon` = sponsor-related on API too

File: to merge with develop [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Merge branch 'develop' of https://github.com/hackmcgill/mchacks7 into feature/7-landingPageBlueprint
```
Comment (loreina) [inline to merge with develop]: what is this file?

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,9 +1,10 @@
 ### Tickets:
+
```
Comment (loreina) [inline .github/pull_request_template.md]: Discard changes for this file

File: README.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ This repository contains the code behind the static site of McHacks 7.
 
 ## Setup
 
-1. Make sure you have [node](https://nodejs.org/en/) and [yarn](https://yarnpkg.com/lang/en/) installed. 
+1. Make sure you have [node](https://nodejs.org/en/) and [yarn](https://yarnpkg.com/lang/en/) installed.
```
Comment (loreina) [inline README.md]: Discard changes for this file

File: src/styles/components/index.js (hunk 2 above)
Comment (loreina) [reply src/styles/components/index.js]: Just gonna make this a bug to fix later

Comment (loreina) [reply src/styles/components/index.js]: HCK-37

**Comment by loreina** [comment]
I fixed a few things. Some stuff that's jank is how the sponsor button is styled with px instead of rem or em, also that it's not divisible by 8 for scaling. We can figure that out later

**Comment by loreina** [comment]
Partially solves HCK-22

File: src/styles/components/index.js (hunk 2 above)
Comment (isabelledelmundo) [reply src/styles/components/index.js]: amazing

**Pull request merged**